src/main/webapp/web.xml

```diff
@@ -2,17 +2,176 @@
+    </servlet-mapping>
+    <!--Registration-->
+    <servlet>
+        <servlet-name>Registration</servlet-name>
+        <servlet-class>com.internet.shop.controllers.RegistrationController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>Registration</servlet-name>
+        <url-pattern>/registration</url-pattern>
+    </servlet-mapping>
+    <!--Delete user-->
+    <servlet>
+        <servlet-name>Delete user</servlet-name>
+        <servlet-class>com.internet.shop.controllers.DeleteUserController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>Delete user</servlet-name>
+        <url-pattern>/users/delete</url-pattern>
+    </servlet-mapping>
+    <!--index-->
+    <servlet>
+        <servlet-name>Add product</servlet-name>
+        <servlet-class>com.internet.shop.controllers.AddProductController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>Add product</servlet-name>
+        <url-pattern>/products/add</url-pattern>
+    </servlet-mapping>
+    <!--All products-->
+    <servlet>
+        <servlet-name>All products</servlet-name>
+        <servlet-class>com.internet.shop.controllers.GetAllProductsController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>All products</servlet-name>
+        <url-pattern>/products/all</url-pattern>
+    </servlet-mapping>
+    <!--Add product to cart-->
+    <servlet>
+        <servlet-name>Add product to cart</servlet-name>
+        <servlet-class>com.internet.shop.controllers.AddProductToShoppingCartController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>Add product to cart</servlet-name>
+        <url-pattern>/shopping-carts/products/add</url-pattern>
+    </servlet-mapping>
+    <!--Edit list of products for Admin-->
+    <servlet>
+        <servlet-name>All products for admin</servlet-name>
+        <servlet-class>com.internet.shop.controllers.admin.GetAllProductsForAdminController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>All products for admin</servlet-name>
+        <url-pattern>/admin/products</url-pattern>
+    </servlet-mapping>
+    <!--Delete product-->
+    <servlet>
+        <servlet-name>Delete product</servlet-name>
+        <servlet-class>com.internet.shop.controllers.DeleteProductController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>Delete product</servlet-name>
+        <url-pattern>/products/delete</url-pattern>
+    </servlet-mapping>
+    <!--Delete product from cart-->
+    <servlet>
+        <servlet-name>Delete product from cart</servlet-name>
+        <servlet-class>com.internet.shop.controllers.DeleteProductFromShoppingCartController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>Delete product from cart</servlet-name>
+        <url-pattern>/shopping-carts/products/delete</url-pattern>
+    </servlet-mapping>
```

Comment:
Better will be "shopping-cart" instead of "carts" because 1 user can have just 1 shopping cart